### ISSUE TITLE
Still unable to list versions with version 0.1.2

### ISSUE THREAD

**Issue by jamesboehmer** [scalaenv/scalaenv#63]
Using 0.1.2 (via homebrew or cloning), I'm still unable to list versions.

```
[james.boehmer:~] $ scalaenv install -l
Available versions:
/usr/local/bin/scalaenv-install: line 51: cd: /usr/local/bin/scalaenv-install/../../share: Not a directory
  Applications
  Desktop
  Documents
  Downloads
  Dropbox
  Library
  Movies
  Music
  Pictures
  Public
  VirtualBox VMs
  bin
  src
  venv
[james.boehmer:~] $ SCALAENV_DEBUG=1 scalaenv install -l
++ [scalaenv:22] type -p greadlink readlink
++ [scalaenv:22] head -1
+ [scalaenv:22] READLINK=/usr/local/bin/greadlink
+ [scalaenv:23] '[' -z /usr/local/bin/greadlink ']'
+ [scalaenv:55] '[' -z '' ']'
+ [scalaenv:56] SCALAENV_ROOT=/Users/james.boehmer/.scalaenv
+ [scalaenv:60] export SCALAENV_ROOT
+ [scalaenv:63] '[' -z '' ']'
++ [scalaenv:64] pwd
+ [scalaenv:64] SCALAENV_DIR=/Users/james.boehmer
+ [scalaenv:73] export SCALAENV_DIR
+ [scalaenv:76] shopt -s nullglob
++ [scalaenv:79] abs_dirname /usr/local/bin/scalaenv
+++ [scalaenv:41] pwd
++ [scalaenv:41] local cwd=/Users/james.boehmer
++ [scalaenv:42] local path=/usr/local/bin/scalaenv
++ [scalaenv:44] '[' -n /usr/local/bin/scalaenv ']'
++ [scalaenv:45] cd /usr/local/bin
++ [scalaenv:46] local name=scalaenv
+++ [scalaenv:47] resolve_link scalaenv
+++ [scalaenv:37] /usr/local/bin/greadlink scalaenv
++ [scalaenv:47] path=../Cellar/scalaenv/0.1.2/bin/scalaenv
++ [scalaenv:44] '[' -n ../Cellar/scalaenv/0.1.2/bin/scalaenv ']'
++ [scalaenv:45] cd ../Cellar/scalaenv/0.1.2/bin
++ [scalaenv:46] local name=scalaenv
+++ [scalaenv:47] resolve_link scalaenv
+++ [scalaenv:37] /usr/local/bin/greadlink scalaenv
++ [scalaenv:47] path=../libexec/scalaenv
++ [scalaenv:44] '[' -n ../libexec/scalaenv ']'
++ [scalaenv:45] cd ../libexec
++ [scalaenv:46] local name=scalaenv
+++ [scalaenv:47] resolve_link scalaenv
+++ [scalaenv:37] /usr/local/bin/greadlink scalaenv
+++ [scalaenv:47] true
++ [scalaenv:47] path=
++ [scalaenv:44] '[' -n '' ']'
++ [scalaenv:50] pwd
++ [scalaenv:51] cd /Users/james.boehmer
+ [scalaenv:79] bin_path=/usr/local/Cellar/scalaenv/0.1.2/libexec
+ [scalaenv:83] export 'PATH=/usr/local/Cellar/scalaenv/0.1.2/libexec:/Users/james.boehmer/.scalaenv/shims:/Users/james.boehmer/.jenv/shims:/Users/james.boehmer/.nodenv/shims:/usr/local/Cellar/pyenv-virtualenv/1.1.3/shims:/Users/james.boehmer/.pyenv/shims:~/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Applications/Wireshark.app/Contents/MacOS'
+ [scalaenv:83] PATH='/usr/local/Cellar/scalaenv/0.1.2/libexec:/Users/james.boehmer/.scalaenv/shims:/Users/james.boehmer/.jenv/shims:/Users/james.boehmer/.nodenv/shims:/usr/local/Cellar/pyenv-virtualenv/1.1.3/shims:/Users/james.boehmer/.pyenv/shims:~/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Applications/Wireshark.app/Contents/MacOS'
+ [scalaenv:86] hook_path=:/Users/james.boehmer/.scalaenv/scalaenv.d:/usr/local/etc/scalaenv.d:/etn/scalaenv.d:/usr/lib/scalaenv/hooks
+ [scalaenv:90] export SCALAENV_HOOK_PATH=:/Users/james.boehmer/.scalaenv/scalaenv.d:/usr/local/etc/scalaenv.d:/etn/scalaenv.d:/usr/lib/scalaenv/hooks
+ [scalaenv:90] SCALAENV_HOOK_PATH=:/Users/james.boehmer/.scalaenv/scalaenv.d:/usr/local/etc/scalaenv.d:/etn/scalaenv.d:/usr/lib/scalaenv/hooks
+ [scalaenv:93] shopt -u nullglob
+ [scalaenv:95] command=install
+ [scalaenv:96] case "${command}" in
++ [scalaenv:109] command -v scalaenv-install
+ [scalaenv:109] command_path=/usr/local/bin/scalaenv-install
+ [scalaenv:110] '[' -n /usr/local/bin/scalaenv-install ']'
+ [scalaenv:112] shift 1
+ [scalaenv:113] '[' -l = --help ']'
+ [scalaenv:120] exec /usr/local/bin/scalaenv-install -l
+ [scalaenv-install:17] export SCALA_BUILD_DEFINITIONS=/Users/james.boehmer/.scalaenv/plugins/scala-install/share
+ [scalaenv-install:17] SCALA_BUILD_DEFINITIONS=/Users/james.boehmer/.scalaenv/plugins/scala-install/share
+ [scalaenv-install:19] '[' -z /Users/james.boehmer/.scalaenv ']'
++ [scalaenv-install:24] type -p greadlink readlink
++ [scalaenv-install:24] head -1
+ [scalaenv-install:24] READLINK=/usr/local/bin/greadlink
+ [scalaenv-install:25] '[' -z /usr/local/bin/greadlink ']'
+ [scalaenv-install:31] '[' -l = --complete ']'
++ [scalaenv-install:40] scala-build --lib
+ [scalaenv-install:40] eval 'lib ()
{
    function parse_options ()
    {
        OPTIONS=();
        ARGUMENTS=();
        local arg option index;
        for arg in "$@";
        do
            if [ "${arg:0:1}" = "-" ]; then
                if [ "${arg:1:1}" = "-" ]; then
                    OPTIONS[${#OPTIONS[*]}]="${arg:2}";
                else
                    index=1;
                    while option="${arg:$index:1}"; do
                        [ -n "$option" ] || break;
                        OPTIONS[${#OPTIONS[*]}]="$option";
                        index=$(($index+1));
                    done;
                fi;
            else
                ARGUMENTS[${#ARGUMENTS[*]}]="$arg";
            fi;
        done
    };
    if [ "$1" == "--$FUNCNAME" ]; then
        declare -f "$FUNCNAME";
        echo "$FUNCNAME \"\$1\";";
        exit;
    fi
}
lib "$1";'
++ [scalaenv-install:71] lib -l
++ [scalaenv-install:65] '[' -l == --lib ']'
+ [scalaenv-install:73] unset FORCE
+ [scalaenv-install:74] unset VERSION
+ [scalaenv-install:76] parse_options -l
+ [scalaenv-install:44] OPTIONS=()
+ [scalaenv-install:45] ARGUMENTS=()
+ [scalaenv-install:46] local arg option index
+ [scalaenv-install:47] for arg in '"$@"'
+ [scalaenv-install:49] '[' - = - ']'
+ [scalaenv-install:50] '[' l = - ']'
+ [scalaenv-install:53] index=1
+ [scalaenv-install:54] option=l
+ [scalaenv-install:55] '[' -n l ']'
+ [scalaenv-install:56] OPTIONS[${#OPTIONS[*]}]=l
+ [scalaenv-install:57] index=2
+ [scalaenv-install:54] option=
+ [scalaenv-install:55] '[' -n '' ']'
+ [scalaenv-install:55] break
+ [scalaenv-install:77] for option in '"${OPTIONS[@]}"'
+ [scalaenv-install:78] case "$option" in
+ [scalaenv-install:83] echo 'Available versions:'
Available versions:
+ [scalaenv-install:84] definitions
+ [scalaenv-install:84] indent
+ [scalaenv-install:70] sed 's/^/  /'
++ [scalaenv-install:66] abs_dirname /usr/local/bin/scalaenv-install/../../share/
+++ [scalaenv-install:47] pwd
++ [scalaenv-install:47] local cwd=/Users/james.boehmer
++ [scalaenv-install:48] local path=/usr/local/bin/scalaenv-install/../../share/
++ [scalaenv-install:50] '[' -n /usr/local/bin/scalaenv-install/../../share/ ']'
++ [scalaenv-install:51] cd /usr/local/bin/scalaenv-install/../../share
/usr/local/bin/scalaenv-install: line 51: cd: /usr/local/bin/scalaenv-install/../../share: Not a directory
++ [scalaenv-install:52] local name=
+++ [scalaenv-install:53] resolve_link ''
++++ [scalaenv-install:43] type -p greadlink readlink
++++ [scalaenv-install:43] head -1
+++ [scalaenv-install:43] /usr/local/bin/greadlink ''
+++ [scalaenv-install:53] true
++ [scalaenv-install:53] path=
++ [scalaenv-install:50] '[' -n '' ']'
++ [scalaenv-install:56] pwd
++ [scalaenv-install:57] cd /Users/james.boehmer
+ [scalaenv-install:66] ls -1 /Users/james.boehmer
  Applications
  Desktop
  Documents
  Downloads
  Dropbox
  Library
  Movies
  Music
  Pictures
  Public
  VirtualBox VMs
  bin
  src
  venv
+ [scalaenv-install:85] exit
```

**Comment by mazgi** [comment]
Thank you for your report.
I'll release a new version.

**Comment by jamesboehmer** [comment]
I see it in the install path, but not in `~/.scalaenv`:

```
[james.boehmer:/usr/local/Cellar/scalaenv/0.1.2] $ ls
total 32
drwxr-xr-x  12 james.boehmer  admin   384 Jan 23 11:32 ./
drwxr-xr-x   3 james.boehmer  admin    96 Jan 23 11:32 ../
drwxr-xr-x   3 james.boehmer  admin    96 Jan 23 11:32 .brew/
-rw-r--r--   1 james.boehmer  admin   580 Jan 23 11:32 INSTALL_RECEIPT.json
-rw-r--r--   1 james.boehmer  admin  1083 Sep 10 00:19 LICENSE
-rw-r--r--   1 james.boehmer  admin  4700 Sep 10 00:19 README.md
drwxr-xr-x   6 james.boehmer  admin   192 Jan 23 11:32 bin/
drwxr-xr-x   5 james.boehmer  admin   160 Sep 10 00:19 completions/
drwxr-xr-x   3 james.boehmer  admin    96 Sep 10 00:19 default-plugins/
drwxr-xr-x  27 james.boehmer  admin   864 Jan 23 11:32 libexec/
lrwxr-xr-x   1 james.boehmer  admin    35 Jan 23 11:32 plugins@ -> /usr/local/var/lib/scalaenv/plugins
lrwxr-xr-x   1 james.boehmer  admin    36 Jan 23 11:32 versions@ -> /usr/local/var/lib/scalaenv/versions
[james.boehmer:/usr/local/Cellar/scalaenv/0.1.2] $ cd plugins
[james.boehmer:/usr/local/Cellar/scalaenv/0.1.2/plugins] $ ls
total 0
drwxr-xr-x  3 james.boehmer  admin   96 Jan 22 12:45 ./
drwxr-xr-x  4 james.boehmer  admin  128 Sep 27 10:33 ../
lrwxr-xr-x  1 james.boehmer  admin   63 Jan 22 12:45 scala-install@ -> ../../../../Cellar/scalaenv/0.1.2/default-plugins/scala-install
[james.boehmer:/usr/local/Cellar/scalaenv/0.1.2/plugins] $ cd ~/.scalaenv/
[james.boehmer:~/.scalaenv] $ ls
total 0
drwxr-xr-x   4 james.boehmer  staff   128 Jan 28 09:41 ./
drwxr-xr-x+ 70 james.boehmer  staff  2240 Jan 28 09:41 ../
drwxr-xr-x   2 james.boehmer  staff    64 Jan 28 09:43 shims/
drwxr-xr-x   2 james.boehmer  staff    64 Jan 28 09:41 versions/
[james.boehmer:~/.scalaenv] $
```

**Comment by 3tty0n** [comment]
This issue was solved by #64. Sorry for your inconvenience in version 0.1.2.
